src/components/satellites-drawer.tsx

```diff
@@ -67,9 +67,9 @@ export const SatellitesDrawer = ({ open }: SatellietesDrawerProps) => {
     <Drawer open={open}>
       <h2 className="pb-6 text-center text-2xl">Manage Satellites</h2>
       {satellites.length === 0 && !form ? (
-        <div className="flex flex-1 flex-col items-center justify-center gap-8 py-4 pb-32">
-          <MdSensorsOff className="h-32 w-32 fill-white/25" />
-          <span className="text-xl">
+        <div className="flex flex-1 flex-col items-center justify-center text-center gap-8 py-4 pb-32">
+          <MdSensorsOff className="h-32 w-32 fill-white/25 " />
+          <span className="text-xl ">
```

Comment:
Reduntant space characters at the end of the `className` strings. Also what is the purpose of this `text-center`? Removing it makes no difference on my end.

```suggestion
        <div className="flex flex-1 flex-col items-center justify-center text-center gap-8 py-4 pb-32">
          <MdSensorsOff className="h-32 w-32 fill-white/25" />
          <span className="text-xl">
```